apps/api/src/hyp3_api/validation.py

```diff
@@ -93,6 +93,13 @@ def check_valid_polarizations(granule_metadata):
         raise GranuleValidationError(f'Only VV and HH polarizations are currently supported, got: {ref_polarization}')
 
 
+def check_not_antimeridian(granule_metadata):
+    for granule in granule_metadata:
+        bbox = granule['polygon'].bounds
+        if abs(bbox[0] - bbox[2]) > 180.0 and bbox[0] * bbox[2] < 0.0:
+            raise GranuleValidationError(f'Granule {granule["name"]} crosses the antimeridian')
```

Comment:
A slightly more explicit error message would be worthwhile, e.g. "Granule {} crossing the antimeridian is not currently supported"

Reply:
Hopefully the new version works for you.